src/App.test.tsx

```diff
@@ -85,4 +85,14 @@ describe("<App />", () => {
 
     expect(await screen.findByText("problemExtension")).toBeInTheDocument();
   });
+
+  it("doesn't show problem detailed page after clicking on search result tile not clickable item on large dekstop", async () => {
+    testRenderer(<App />);
+    const author = (await screen.findByText(
+      "Maria Konopnicka"
+    )) as HTMLInputElement;
+    userEvent.click(author);
+
+    expect(await screen.findByText(/searchResults/)).toBeInTheDocument();
+  });
```

Comment:
I'd test it in cypress, specifying exactly the tool : >

Reply:
I've moved it to cypress, by default it tests on screen with dimension 1000x600, so I haven't specified the tool.